test/e2e/layers/layerLabel.test.js

```diff
@@ -0,0 +1,40 @@
+import { test, expect, chromium } from '@playwright/test';
+
+test.describe('Layer Label Tests', () => {
+ 
+	let page;
+	let context;
+	test.beforeAll(async function () {
+		context = await chromium.launchPersistentContext('');
+		page =
+			context.pages().find((page) => page.url() === 'about:blank') ||
+			(await context.newPage());
+		await page.goto('layerLabel.html');
+	});
+
+	test.afterAll(async function () {
+		await context.close();
+	});
+
+	test('Name of unnamed layer is Layer', async () => {
+		const map = await page.locator('body > mapml-viewer');
+		await map.evaluate((map) =>
+			map.querySelector('layer-').removeAttribute('hidden')
+		);
+
```

Comment:
This is not needed, as the layer does not have a "hidden" attribute, I think we copied this over from a previous test.